testing/nodejs/WORKSPACE.bzlmod

```diff
@@ -0,0 +1,14 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# build_bazel_rules_nodejs provides no Bazel module and is not listed on the
+# Bazel Central Registry. The rule set is deprecated in favor of rules_js, so,
+# we cannot expect a Bazel module to be created for it.
+
+# TODO[AH] Migrate rules_nixpkgs_nodejs to use rules_js for any JavaScript
+# targets, rules_nodejs can still be used for the core functionality, the
+# toolchain in particular.
+http_archive(
+    name = "build_bazel_rules_nodejs",
+    sha256 = "dcc55f810142b6cf46a44d0180a5a7fb923c04a5061e2e8d8eb05ccccc60864b",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.8.0/rules_nodejs-5.8.0.tar.gz"],
+)
```

Comment:
@aherrmann 
AFAIR this dependency is used only to obtain a single variable (I think it was `PLATFORMS` or something). One option to get rid of this is to hardcode those platforms that we're interested in. Downside of this solution is the need of keeping it synchronized (without the a priori knowledge that something got changed). For it's ~ok to accept it :)

Reply:
`PLATFORMS` [comes from `@rules_nodejs`](https://github.com/tweag/rules_nixpkgs/blob/master/toolchains/nodejs/nodejs.bzl#L3), which corresponds to `https://github.com/bazelbuild/rules_nodejs/releases/download/5.8.0/rules_nodejs-core-5.8.0.tar.gz`. That's separate from `build_bazel_rules_nodejs`. This dependency here is needed for the [`nodejs_test` target](https://github.com/tweag/rules_nixpkgs/pull/332/files#diff-4bd2f834ad36fd326dedd9619e1d7873e0542173559c844809b46e7cf761990dR6) under `testing/nodejs/tests`.